scripts/publish.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+CPU_ARCHS="amd64 arm64 arm ppc64le s390x"
+TAG_COMMIT=$(git rev-list --abbrev-commit --tags --max-count=1)
+COMMIT=$(git rev-parse --short HEAD)
+TAG=$(git describe --abbrev=0 --tags ${TAG_COMMIT} 2>/dev/null || true)
+VERSION="${TAG}"
+
+# if the current commit, does not correspond to a tag, create a verbose version string.
+if [ "${TAG_COMMIT}" != "${COMMIT}" ]; then
+  VERSION=$(git rev-parse --abbrev-ref HEAD | tr / -)-$(date +%Y-%m-%d)-$(git rev-parse --short HEAD)
+fi
+
+# build and push arch specific images
+for arch in ${CPU_ARCHS}; do
+  DOCKER_REPO="${QUAY_PATH}" GOARCH="${arch}" make container
```

Comment:
QUAY_PATH is not set in the script, but there is `set -u` on top of the script which means script should fail to execute.

Reply:
QUAY_PATH is injected by an environment variable from with CI, but we can set a default here as well.